backend/fms_core/resources/_generic.py

```diff
@@ -2,14 +2,40 @@
 
 from import_export import resources
 from import_export.results import RowResult
+from django.core.exceptions import ValidationError
 from reversion.models import Version
 
 
 class GenericResource(resources.ModelResource):
-    clean_model_instances = True
-    skip_unchanged = True
     ERROR_CUTOFF = 20
     errorCount = 0
+    manuallyExclude = []
```

Comment:
errorCount and manuallyExclude.. camelCase is for JS, python uses snake_case :P also I would change manuallyExclude to a more meaningful name :)